apps/summon-safe-app/project.json

```diff
@@ -0,0 +1,92 @@
+{
+  "root": "apps/summon-safe-app",
+  "sourceRoot": "apps/summon-safe-app/src",
+  "projectType": "application",
+  "targets": {
+    "build": {
+      "executor": "@nxext/vite:build",
+      "outputs": ["{options.outputPath}"],
+      "defaultConfiguration": "production",
+      "options": {
+        "outputPath": "dist/apps/summon-safe-app",
+        "baseHref": "/",
+        "configFile": "@nxext/vite/plugins/vite",
+        "frameworkConfigFile": "@nxext/react/plugins/vite"
+      },
+      "configurations": {
+        "production": {
+          "fileReplacements": [
+            {
+              "replace": "apps/summon-safe-app/src/environments/environment.ts",
+              "with": "apps/summon-safe-app/src/environments/environment.prod.ts"
+            }
+          ],
+          "optimization": true,
+          "outputHashing": "all",
+          "sourceMap": false,
+          "namedChunks": false,
+          "extractLicenses": true,
+          "vendorChunk": false
+        }
+      }
+    },
+    "serve": {
+      "executor": "@nxext/vite:dev",
+      "options": {
+        "outputPath": "dist/apps/summon-safe-app",
+        "baseHref": "/",
+        "configFile": "@nxext/vite/plugins/vite",
+        "frameworkConfigFile": "@nxext/react/plugins/vite"
+      },
+      "configurations": {
+        "production": {
+          "buildTarget": "summon-safe-app:build:production",
+          "hmr": false
+        }
+      }
+    },
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["apps/summon-safe-app/**/*.{ts,tsx,js,jsx}"]
+      }
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": ["coverage/apps/summon-safe-app"],
+      "options": {
+        "jestConfig": "apps/summon-safe-app/jest.config.js",
+        "passWithNoTests": true
+      }
+    },
+    "tsc": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "command": "tsc --project apps/summon-app/tsconfig.app.json --noEmit --skipLibCheck",
+        "color": true
+      }
+    },
+    "deploy": {
+      "executor": "@alexkeating/nx-skynet:deploy",
+      "options": {
+        "uploadDir": "dist/apps/summon-safe-app",
+        "registrySeedVarName": "DAOHAUS_SUMMON_SKYNET_SEED",
```

Comment:
We need to create a new seed and env variable for the safe summon app as well as in the deployStaging command.

Reply:
Just created the seed and env variable:

DAOHAUS_SUMMON_SAFE_SKYNET_SEED